diff -u Additions/Make Additions.new/Make
--- Additions/Make	2012-01-05 07:38:42.000000000 +0900
+++ Additions.new/Make	2012-09-10 09:39:43.466771900 +0900
@@ -28,13 +28,14 @@
 extract_hs.v
 extract_scm.v
 # the test :
--custom "$(CAMLBIN)ocamlopt -o fib nums.cmxa fibo.mli fibo.ml main.ml" extract.vo fib
+-custom "$(CAMLBIN)ocamlopt -o fib.exe nums.cmxa fibo.mli fibo.ml main.ml" extract.vo fib.exe
 -custom
 	"@echo '***** test: checking Fibonacci(2000) *****'
 	rm -f .fib_2000
 	./fib 2000 > .fib_2000	
+	dos2unix .fib_2000
 	cat .fib_2000
 	cmp .fib_2000 fib_2000
 	@echo '************** End of test ****************'"
-  fib test
+  fib.exe test
 # $Id: Make 1059 2009-11-12 18:26:29Z serpyc $
