diff -ru MapleMode/fake_maple/Makefile MapleMode.new/fake_maple/Makefile
--- MapleMode/fake_maple/Makefile	2012-01-05 07:38:18.000000000 +0900
+++ MapleMode.new/fake_maple/Makefile	2012-09-13 18:28:24.251742800 +0900
@@ -1,9 +1,9 @@
-all: fake_maple
+all: fake_maple.exe
 
-fake_maple: fake_maple.ml
-	$(CAMLBIN)ocamlc -o fake_maple str.cma fake_maple.ml
+fake_maple.exe: fake_maple.ml4
+	$(CAMLBIN)ocamlc -o fake_maple.exe str.cma fake_maple.ml4
 
 clean:
-	rm -f fake_maple fake_maple.cmo fake_maple.cmi
+	rm -f fake_maple.exe fake_maple.cmo fake_maple.cmi
 
 .PHONY: all
diff -ru MapleMode/Make MapleMode.new/Make
--- MapleMode/Make	2012-01-05 07:38:19.000000000 +0900
+++ MapleMode.new/Make	2012-09-13 18:28:51.874750500 +0900
@@ -3,7 +3,9 @@
 COQC = "export MAPLE=./fake_maple/fake_maple ; $(COQBIN)coqc"
 Examples.v
 Maple.v
-maple.ml
-fake_maple/fake_maple.ml
--custom "$(CAMLLINK) -o $@ str.cma $<" "fake_maple/fake_maple.cmo" "fake_maple/fake_maple"
--custom "$(COQC) $(COQDEBUG) $(COQFLAGS) $<" "Maple.v fake_maple/fake_maple" "Maple.vo"
\ ファイル末尾に改行がありません
+maple.ml4
+-custom "cp $< $@" maple.ml maple.ml4
+-custom "cp $< $@" fake_maple/fake_maple.ml fake_maple/fake_maple.ml4
+fake_maple/fake_maple.ml4
+-custom "$(CAMLLINK) -o $@ str.cma $<" "fake_maple/fake_maple.cmo" "fake_maple/fake_maple.exe"
+-custom "$(COQC) $(COQDEBUG) $(COQFLAGS) $<" "Maple.v fake_maple/fake_maple.exe" "Maple.vo"
diff -ru MapleMode/maple.ml MapleMode.new/maple.ml
--- MapleMode/maple.ml	2012-04-27 09:00:15.000000000 +0900
+++ MapleMode.new/maple.ml	2012-09-13 18:42:02.167104900 +0900
@@ -298,7 +298,7 @@
     let inc = open_in tmp in
     let exp =
       try Grammar.Entry.parse mexpr_s (Stream.of_channel inc)
-      with Stdpp.Exc_located (_,e) -> raise e in
+      with Ploc.Exc (_,e) -> raise e in
     begin
       close_in inc;
       Sys.remove tmp;
