diff -ruN QuicksortComplexity/Make QuicksortComplexity.new/Make
--- QuicksortComplexity/Make	1970-01-01 09:00:00.000000000 +0900
+++ QuicksortComplexity.new/Make	2012-09-13 22:07:40.901925200 +0900
@@ -0,0 +1,37 @@
+-R . QuicksortComplexity
+arith_lems.v
+expec.v
+fix_measure_utils.v
+harmonic.v
+indices.v
+insertion_sort.v
+list_length_expec.v
+list_utils.v
+monads.v
+monoid_expec.v
+monoid_monad_trans.v
+monoid_tree_monad.v
+nat_below.v
+nat_seqs.v
+NDP.v
+ne_list.v
+ne_tree.v
+ne_tree_monad.v
+qs_case_split.v
+qs_cases.v
+qs_CM_U_expec_cost_eq.v
+qs_cmp_prob.v
+qs_correct.v
+qs_definitions.v
+qs_det_avg_complexity.v
+qs_det_parts.v
+qs_nondet_avg_complexity.v
+qs_parts.v
+qs_sound_cmps.v
+qs_worst.v
+skip_list.v
+sort_order.v
+sums_and_averages.v
+U.v
+util.v
+vec.v
