diff -ru Rational/Make Rational.new/Make
--- Rational/Make	2012-01-05 07:38:37.000000000 +0900
+++ Rational.new/Make	2012-09-13 22:21:18.013685200 +0900
@@ -4,14 +4,17 @@
 ./Natural/nat.v
 ./Rewrite/LeibnizRewrite/HS/hS.ml
 ./Rewrite/LeibnizRewrite/HS/hS_tac.ml
+-custom "cp $< $@" ./Rewrite/LeibnizRewrite/HS/hS_tac.ml ./Rewrite/LeibnizRewrite/HS/hS_tac.ml4
 ./Rewrite/LeibnizRewrite/AC/aC.ml
-./Rewrite/LeibnizRewrite/AC/aC_tac.ml
+./Rewrite/LeibnizRewrite/AC/aC_tac.ml4
+-custom "cp $< $@" ./Rewrite/LeibnizRewrite/AC/aC_tac.ml ./Rewrite/LeibnizRewrite/AC/aC_tac.ml4
 ./Rewrite/LeibnizRewrite/MLstuff/sort_tac.ml
 ./Rewrite/LeibnizRewrite/MLstuff/struct.ml
 ./Rewrite/GenericRewrite/genMLstuff/frame.ml
 ./Rewrite/GenericRewrite/genMLstuff/basic.ml
 ./Rewrite/GenericRewrite/HeadSimpl/main.ml
 ./Rewrite/GenericRewrite/HeadSimpl/tacentry.ml
+-custom "cp $< $@" ./Rewrite/GenericRewrite/HeadSimpl/tacentry.ml ./Rewrite/GenericRewrite/HeadSimpl/tacentry.ml4
 ./Rewrite/GenericRewrite/HeadSimpl/HeadSimpl.v
 ./Rewrite/LeibnizRewrite/HS/HS.v
 ./Rewrite/LeibnizRewrite/AC/AC.v
